src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
```

Comment:
It should be `stars` elements, so it should be named as `.stars__star` if you use BEM methodology.